DashKit/DashPumpManager.swift

```diff
@@ -1605,12 +1605,15 @@ extension DashPumpManager: PodCommManagerDelegate {
             let content = Alert.Content(title: LocalizedString("Pod System Error", comment: "Alert title for Pod System Error"),
                                         body: error.localizedDescription,
                                         acknowledgeActionButtonLabel: LocalizedString("OK", comment: "Alert acknowledgment OK button"))
-            let parameters = (try? JSONEncoder().encode(error)).flatMap { String(data: $0, encoding: .utf8) }
+            let metadata: Alert.Metadata = [
```

Comment:
See my comment in your `LoopKit` [PR](https://github.com/tidepool-org/LoopKit/pull/438/files#r804952012) ... it may make this a little less verbose.

Reply:
See my response to said comment.